music21/musicxml/xmlToM21.py

```diff
@@ -3881,11 +3881,18 @@ def xmlTechnicalToArticulation(self, mxObj):
             if tag in ('heel', 'toe'):
                 if mxObj.get('substitution') is not None:
                     tech.substitution = xmlObjects.yesNoToBoolean(mxObj.get('substitution'))
+            if tag == 'bend':
+                tech.bendAlter = interval.Interval(int(mxObj.find('bend-alter').text))
+                if mxObj.find('pre-bend') is not None:
+                    tech.preBend = True
+                if mxObj.find('release') is not None:
+                        tech.release = int(mxObj.find('release').get('offset'))
```

Comment:
as above -- offset needs to be converted to music21 quarterLengths.

except clause without Try.

Reply:
Both fixed with https://github.com/cuthbertLab/music21/pull/1580/commits/58eeb04646b04f5c0501795a30aa67e7083b23e6